### DISCUSSION TITLE
shorten the overhead to delete a tour

### DISCUSSION THREAD

**Discussion by Marty56** [mytourbook/mytourbook#1211] [Ideas]
when deleting a tour the overhead for the user is super annoying and really bad usability.
the user has to click several times until the process is completed.

The process (sorry I only have the German version )

1. Rechtsklick auf den Track
2. Touren löschen
3. Untermenu "Touren löschen"
4. "Ausgewählte Touren löschen" anklicken
5. Fenster mit "Sind sie sicher, die ausgewählten Touren löschen"
6. noch ein Fenster "Sind sie wirklich sicher 1 Tour zu ..."

in total 6 Interaktion!! This is insane!!

I wish a shortcut character, that deletes a tour with our further questions.

**Comment by FJBDev** [comment]
I had a similar thought when I discovered MT. My understanding is that it's to make it harder for a user to delete a tour by accident.

**Reply by Thompson-ongithub** [reply]
That's my understanding as well. It never really bothered me since I'm not often deleting (and then reimporting) tours. It might seem a bit paranoid to do so, but since a deleted tour cannot be recovered my reckoning was "better save then sorry". You can't delete a tour by accident (so I acutally prefer more than one confirmation window)

**Comment by wolfgang-ch** [comment]
When using the mouse it can be annoying but when using the keyboard it needs only 4 different keys which is my preferred way.

Keys for the German language to delete a tour:

1. `<Shift><F10>` opens the context menu
2. `<l>` 3x
3. `<Enter>` 2x

or

1. `<Ctrl><Shift><Del>`
2. `<Enter>` 2x

**Comment by Marty56** [comment]
I am on a Mac.
The first workaround is working, the second is not working because there is no del key on the max. 

Ideally it would be just the del key alone. On the Mac it then would be fn Backspace.
And after this key only one confirmation window.

**Reply by wolfgang-ch** [reply]
The next version will also support the `<Ctrl><Shift><Backspace>` keys additional to the `<Ctrl><Shift><Del>` key https://github.com/mytourbook/mytourbook/commit/ef1d839c4328dbc0235afbc52f461b24e3462df6

**Comment by Marty56** [comment]
Great!

**Comment by FJBDev** [comment]
> The next version will also support the `<Ctrl><Shift><Backspace>` keys additional to the `<Ctrl><Shift><Del>` key [ef1d839](https://github.com/mytourbook/mytourbook/commit/ef1d839c4328dbc0235afbc52f461b24e3462df6)

I have updated the CheatSheet to include both of them
[MT-CheatSheet-23.10.pdf](https://github.com/mytourbook/mytourbook/files/13046424/MT-CheatSheet-23.10.pdf)

**Comment by wolfgang-ch** [comment]
![image](https://github.com/mytourbook/mytourbook/assets/1283445/031fc1f8-9647-4509-b4fc-0b2e3568ea56)

**Comment by Marty56** [comment]
Reduction of the number of confirmation window to only one would also desirable.

**Reply by wolfgang-ch** [reply]
It is not planned to reduce the necessary steps when deleting a tour. A tour can be deleted very quickly by using the keyboard shortcuts.
